index.js

```diff
@@ -311,7 +311,8 @@ class MongoTenant {
         continue;
       }
 
-      MongoTenantModel[staticProperty] = BaseModel[staticProperty];
+      let descriptor = Object.getOwnPropertyDescriptor(BaseModel, staticProperty)
```

Comment:
Please add a semicolon to the end of line, so that the codebase stays consistent. Thx